### DISCUSSION TITLE
Question on Troubleshooting of a Blast! This Feed has Encountered a problem...

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#5704] [Q&A] [answered]
So I have a question as I continue to play around with XPath and Web Scrapping.  As I'm working on a new site to scrape, I get the "Blast! This feed has encountered a problem" and I was wondering how to best troubleshoot this.  Is there more than just the http connection logs (in the latest Docker image build) that would help me to better know why this newly created feed may have a problem.  Thanks for any pointers or advice.  

Site I'm Working with Image:
![Screenshot 2023-10-12 at 3 42 53 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/c207399e-c7b8-411b-9cff-c2935d88dafb)

XPath Settings (which do not show an issue)
![Screenshot 2023-10-12 at 3 43 12 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/4dcad1c2-088e-4578-8edb-3b793b251158)

**Comment by Frenzie** [comment, marked as answer]
There's also Settings (top right cog) → Logs, unless that's what you meant.

**Comment by quella** [comment]
I actually was not aware of that Log view, but was referring to the Docker Log command.  :)  

When I look at the Cog -> Log, it seems to show a Error 403 Forbidden.  Wonder if they look for web scraping and block that type of activity on the site.  Interesting.



Log Level | Timestamp | Log Message
  | 2023-10-12 16:00:02 | HTML+XPath Web scraping failed for [https://www.artstation.com/?sort_by=latest&amp;dimension=all]
  | 2023-10-12 16:00:02 | Error fetching content: HTTP code 403: https://www.artstation.com/?sort_by=latest&dimension=all

**Reply by Frenzie** [reply]
You can distinguish between whether it's the IP or the user agent that's blocked with e.g. curl.

**Reply by Frenzie** [reply]
Something in the general direction of this with and without the -A
```
curl -v -A "FreshRSS/1.21.0" 'https://www.artstation.com?sort_by=laster&dimension=all'
```

**Reply by quella** [reply]
I will give this a try from the same host from the command line and see what comes up.  Thanks for the hints and direction.

**Reply by quella** [reply]
It is interesting as I see this in the CURL logs.  Seems to be some issue with HTTP/2 and the 403.  More research needed.  

* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 100)!
< HTTP/2 403 
< date: Thu, 12 Oct 2023 20:32:45 GMT
